src/main/kotlin/io/emeraldpay/dshackle/upstream/ethereum/EthereumLocalReader.kt

```diff
@@ -59,20 +58,15 @@ class EthereumLocalReader(
             // we do not want to serve any requests (except hardcoded) that have nonces from cache
             return Mono.empty()
         }
-        val common = commonRequests(key)
-            ?.switchIfEmpty { Mono.just(nullValue to null) }
```

Comment:
Why is it removed? In such case the `executeOnRemote` branch will always be executed, but we wanted to avoid it